xc-admin/packages/xc-admin-common/src/governance_payload/ExecutePostedVaa.ts

```diff
@@ -75,11 +75,21 @@ export class ExecutePostedVaa implements PythGovernanceAction {
   }
 
   /** Decode ExecutePostedVaa */
-  static decode(data: Buffer): ExecutePostedVaa {
+  static decode(data: Buffer): ExecutePostedVaa | undefined {
     let header = PythGovernanceHeader.decode(data);
-    let deserialized = this.layout.decode(
-      data.subarray(PythGovernanceHeader.span)
-    );
+    if (!header) {
+      return undefined;
+    }
+
+    let deserialized;
+    try {
+      deserialized = this.layout.decode(
+        data.subarray(PythGovernanceHeader.span)
+      );
+    } catch {
+      return undefined;
+    }
+
```

Comment:
buffer-layout can throw errors, so we need to handle them here that way the user of ours structs won't have to deal with them